src/figuresClasses.ts

```diff
@@ -1,11 +1,84 @@
-export interface Figure {}
+export interface Figure {
+  getArea(): number;
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  public shape: string = 'triangle';
+
+  public color: string;
+
+  constructor(
+    color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.color = color;
```

Comment:
[CODE KNOWLEDGE] - When you declare a constructor parameter with an access modifier TypeScript implicitly creates a property on the class with the same name as the parameter and assigns the parameter value to that property. You don't need to do it manually. In your case, you don't need to manually assign the `color` parameter to `this.color` in the constructor. You can declare it as a public field in the constructor parameters list, and TypeScript will do it for you. The same applies to the constructors in the `Circle` and `Rectangle` classes.